package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clin-proxy-api",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
Still good. Version in the dev was bumped to 1.4.1. Your version should still be 1.5.0.